asistente_ladm_col/gui/supplies/snc_data_sources_widget.py

```diff
@@ -185,10 +196,11 @@ def save_settings(self):
 
     def restore_settings(self):
         settings = QSettings()
-        self.txt_file_path_predio_sancion.setText(settings.value('Asistente-LADM-COL/etl_snc/predio_sancion_path', ''))
+        self.txt_file_path_predio_bloqueo.setText(settings.value('Asistente-LADM-COL/etl_snc/predio_sancion_path', ''))
```

Comment:
Why not `Asistente-LADM-COL/etl_snc/predio_bloqueo_path`?

Reply:
Yes, this is better.

Reply:
Shouldn't it be fixed also [here](https://github.com/SwissTierrasColombia/Asistente-LADM-COL/blob/1b4889649f2d1b5f8f4d4e3bec46c3ae5249f4da/asistente_ladm_col/gui/supplies/snc_data_sources_widget.py#L199):         

    self.txt_file_path_predio_bloqueo.setText(settings.value('Asistente-LADM-COL/etl_snc/predio_sancion_path', ''))

Reply:
Yes, in both is necessary, thanks 

Excuse, I didn't change it before.